src/js/utils/option-utils.jsx

```diff
@@ -260,6 +260,21 @@ export const fetchBins = async (
   return response.data.data;
 };
 
+export const fetchLocations = async ({ activityCodes }) => {
```

Comment:
how did it end up? Was it finally be supposed to be a debounced fetch or not?

Reply:
No, it doesn't need to be debounced